options/options.go

```diff
@@ -82,7 +83,13 @@ type TerragruntOptions struct {
 func NewTerragruntOptions(terragruntConfigPath string) *TerragruntOptions {
 	workingDir := filepath.Dir(terragruntConfigPath)
 
-	downloadDir := filepath.Join(os.TempDir(), "terragrunt")
+	user, err := user.Current()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
```

Comment:
I'm not comfortable with a random method exiting. Code is much more predictable if all errors are returned. I'm sure `NewTerragrunOptions` is used in a bunch of places, but if the method can now have an error, it needs to reflect that in its signature.